src/uct/ib/base/ib_md.c

```diff
@@ -1192,6 +1183,22 @@ static ucs_status_t uct_ib_query_md_resources(uct_component_t *component,
     struct ibv_device **device_list;
     ucs_status_t status;
     int i, num_devices;
+    rlim_t memlock_limit;
+
+    status = ucs_sys_get_memlock_rlimit(&memlock_limit);
+    if ((status == UCS_OK) &&
+        (memlock_limit != RLIM_INFINITY) &&
+        (memlock_limit <= (500 * UCS_MBYTE))) {
+        /* Disable the RDMA devices because of too strong locked memory limit*/
```

Comment:
What do you think about not `low` but `strict` limit?

```suggestion
        /* Disable the RDMA devices because of too strict locked memory limit*/
```